src/common/java/net/minecraftforge/gradle/common/util/Utils.java

```diff
@@ -491,4 +497,27 @@ public static String replaceTokens(Map<String, ?> tokens, String value) {
 
         return buf.toString();
     }
+
+    public static void setupIDEResourceCopy(@Nonnull final Project project) {
+        if (project.getPlugins().hasPlugin(IdeaPlugin.class)) {
+            final IdeaPlugin idea = project.getPlugins().getPlugin(IdeaPlugin.class);
+            project.getTasks().create("copyIdeaResources", CopyIDEAResources.class, task -> task.configure(idea.getModel()));
+        } else {
+            project.getLogger().warn("'idea' plugin is not found! IDEA resources will not be copied.");
+            project.getTasks().create("copyIdeaResources", Task.class); // Create dummy class
+        }
+
+        if (project.getPlugins().hasPlugin(EclipsePlugin.class)) {
+            project.getTasks().create("copyEclipseResources", CopyEclipseResources.class, task -> {
+                task.dependsOn("eclipse");
+                project.getTasks().named("eclipse").configure($ -> $.doLast($$ -> {
+                    final EclipseModel eclipse = project.getExtensions().findByType(EclipseModel.class);
+                    task.configure(eclipse, project);
+                }));
+            });
+        } else {
+            project.getLogger().warn("'eclipse' plugin is not found! Eclipse resources will not be copied.");
+            project.getTasks().create("copyEclipseResources", Task.class); // Create dummy class
+        }
```

Comment:
I don't really like warning logs just cuz a plugin is a missing. I am still thinking that it would be nice if we applied these plugins automatically for the user if they enable the property. However, since the property is read pretty late, this is probably not possible, and it would be best to just update the MDK to include the `idea` plugin. So I guess this is fine...